### ISSUE TITLE
[News] Display problem with Headline Template in case of high resolution

### ISSUE THREAD

**Issue by hthabti** [Meeds-io/meeds#2677]
**Steps:**
- open news list portlet on headline template
- using pc with high resolution (3 * the width of the column in which the news is displayed)

example: if news list is opened on 1249 hence the resolution to reproduce the behavior should be 3747

**Current:**
the news list is (on headline template) displayed on mobile version while I am using a desktop view

**Expected:**
calculate the display based on the width of the screen displayed

eXo task: 74804

**Comment by srenault-meeds** [comment]
Please make sure to match the product breakpoints

https://github.com/Meeds-io/social/blob/e8c37f03a52e15bb47db5b4e7c50b9d2a2f6e76c/webapp/src/main/webapp/groovy/social/webui/UISocialPortalApplicationHead.gtmpl#L57

**Comment by margondicco** [comment]
@Jihed525  ? some update here ?

**Comment by Jihed525** [comment]
@margondicco a confirmation is needed from @hthabti if issue still reproduced

**Comment by margondicco** [comment]
Hello it still reproduced for RC5

@srenault-meeds IMO fixed to be link with the fix on template : WDYT ?

**Comment by srenault-meeds** [comment]
Yes would be better indeed to work it as a batch dedicated to newslist templates

**Comment by margondicco** [comment]
@Jihed525 @srenault-meeds could we consider it ?

Thank you

**Comment by Jihed525** [comment]
builders task created 8884 (but it will be tricky on how to reproduce it)

fyi @hthabti

**Comment by hthabti** [comment]
https://github.com/user-attachments/assets/6f3909d0-f304-4d94-bcdc-fc29e6f4bfa1
